workshops/views.py

```diff
@@ -853,6 +853,12 @@ def event_edit(request, event_ident):
                     ),
                 )
 
+                # if event.attendance is lower than number of learners, then
+                # update the attendance
+                learners = event.task_set.filter(role__name='learner').count()
+                Event.objects.filter(pk=event.pk, attendance__lt=learners) \
+                             .update(attendance=learners)
+
```

Comment:
Is this the same logic as in util.py?  Can it be consolidated so that there's one place where we do this calculation and update?  Because it's highly likely to evolve.